py/src/braintrust/framework.py

```diff
@@ -177,32 +184,42 @@ def pluralize(n, singular, plural):
         return plural
 
 
-def report_evaluator_result(eval_name, results, summary, verbose):
+def report_evaluator_result(eval_name, results, summary, verbose, jsonl):
```

Comment:
same here, would be nice if we passed by keyword name or made these keyword-only args.

Reply:
fixed